model_mng/model_hash.py

```diff
@@ -0,0 +1,77 @@
+import hashlib
+import logging
+import os
+import json
+from aiohttp import web
+from pathlib import Path
+
+import os
+import time
+import logging
+import folder_paths
+
+class ModelHash():
+    def __init__(self):
+        self.model_prefix_names = {"ckpt_name": "checkpoints",
+                          "control_net_name": "controlnet",
+                          "lora_name": "loras",
+                          "unet_name":"unet",
+                          "clip_name": "clip",
+                          "style_model_name": "style_models",
+                          "gligen_name": "gligen",
+                          "vae_name": "vae" }
+        pass
+
+
+    def get_MD5_hash(self, path):
+        path = Path(path)
+        path.parent.mkdir(parents=True, exist_ok=True)
+        if not path.exists():
+            return None
+
+        file_digest = hashlib.sha256(open(path, 'rb').read()).hexdigest()
```

Comment:
running a full hash of a model on the fly without caching or anything will be terribly slow